### DISCUSSION TITLE
Artemis image with latest tags not working.

### DISCUSSION THREAD

**Discussion by achinjain214** [arkmq-org/arkmq-org-broker-operator#697] [Q&A]
@brusdev we have tried pulling below images with latest tags but none of them is working. Anyone has any idea on this?

- name: RELATED_IMAGE_ActiveMQ_Artemis_Broker_Init_2290
          value: quay.io/artemiscloud/activemq-artemis-broker-init@sha256:56bae938a6b4cd87f9f5284931f440f9b9d116cda3b63b13f83a4b1c6c641959
        - name: RELATED_IMAGE_ActiveMQ_Artemis_Broker_Kubernetes_2290
          value: quay.io/artemiscloud/activemq-artemis-broker-kubernetes@sha256:4be60cfa2a300b2eef9b5f10ea756bd3f55d75fbd8890232d907f8d64080449b
        - name: RELATED_IMAGE_ActiveMQ_Artemis_Broker_Init_2300
          value: quay.io/artemiscloud/activemq-artemis-broker-init@sha256:88a0098a628fe43a9b422f8ebb774086fb65ed389a195e2779482a4a63447e28
        - name: RELATED_IMAGE_ActiveMQ_Artemis_Broker_Kubernetes_2300
          value: quay.io/artemiscloud/activemq-artemis-broker-kubernetes@sha256:8fbdd1ba728058d5f437186b9055ca28596140068ebc69ecff4fa52717b19a82

**Comment by brusdev** [comment]
@achinjain214 I have just checked all those container images and confirm that they works. Maybe it was a temporary issue because quay.io does not restrict anonymous pulls against its repositories (either public or private) and only rate limits in the most severe circumstances to maintain service levels (e.g. tens of requests per second from the same IP address).

Feel free to reopen this issue if you get this issue again.
